pkg/arvo/sys/vane/jael.hoon

```diff
@@ -882,7 +889,8 @@
       ::  if changing rift upward, then signal a breach
       ::
       =?    ..feel
-          ?&  ?=(^ maybe-point)
+          ?&  (~(has by original) who)
+              ?=(^ maybe-point)
```

Comment:
May or may not want to update the comment (here and on 854) to point out the "and we have older keys for them" clause.